addons/account/models/account_invoice_import_wizard.py

```diff
@@ -11,11 +11,14 @@ def _get_default_journal_id(self):
         return self.env['account.journal'].search([('type', '=', 'purchase')], limit=1)
 
     attachment_ids = fields.Many2many('ir.attachment', string='Files')
-    journal_id = fields.Many2one(string="Journal", comodel_name="account.journal", required=True, domain="[('type', '=', 'purchase')]", default=_get_default_journal_id, help="Journal where to generate the bills")
```

Comment:
@jbm-odoo Better to put the formatting in a separate commit.  Or only do it when you really change something.  Makes it difficult to check what changed.